model/src/main/kotlin/config/CopyrightGarbage.kt

```diff
@@ -28,7 +30,8 @@ data class CopyrightGarbage(
     /**
      * A set of literal strings that identify garbage Copyright findings.
      */
-    val items: SortedSet<String> = sortedSetOf()
+    @JsonSerialize(converter = StringSortedSetConverter::class)
```

Comment:
I'd prefer to use this sort order: https://github.com/oss-review-toolkit/ort/blob/6e4a3f2bb7379eaadf3a9bb01ba321dba53d8dcf/helper-cli/src/main/kotlin/commands/ImportCopyrightGarbageCommand.kt#L68

This is the order you get if you manage the garbage file via helper-cli. I do recall it was deliberately choosen, IIRC it was for consistency with the Unix sort command. Maybe the git history tells it.

Reply:
So, as this seems to be the only place where the `CopyrightGarbage` class is serialized, shouldn't it be safe to simply drop the `@JsonSerialize(converter = StringSortedSetConverter::class)` annotations and keep the `items` data type at a plain `Set`?

Reply:
Hm, as soon as another place is introduced this can easily be missed. 
What do you think about using a converter which converts a `Set` to a `Set` (not to `SortedSet`) which ensures the converted set is sorted with the above `Collator`. So to say, move the sorting from that `Import...Command` to here?

Reply:
To be honest, that case seems to be a bit too special to me to justify the introduction of such a Collator-Set-Converter. I'm fine with having this special sorting in the special `ImportCopyrightGarbageCommand`, but I wouldn't go as far as assuming that every user of `CopyrightGarbage` would want exactly that sorting.

Reply:
> Why isn't that too special then too?

At least to me, it's much more natural to say that strings are sorted alphabetically, or packages are sorted by id, than saying that Copyright garbage items (which also are just strings) are sorted by a collator for the English locale with UTF-8 encoding using the POSIX variant...

Reply:
> I'm fine with having this special sorting in the special ImportCopyrightGarbageCommand, but I wouldn't go as far as assuming that every user of CopyrightGarbage would want exactly that sorting.

Ok, so shall we find a way to not break it?

Reply:
> Ok, so shall we find a way to not change sorting in ImportCopyrightGarbageCommand then?

You mean to find a way for `ImportCopyrightGarbageCommand` to not use the custom `Collator`? Well, I don't care 😸 Like I said, I'm fine with any user of `CopyrightGarbage` that requires custom sorting to apply that sorting outside of `CopyrightGarbage`, like `ImportCopyrightGarbageCommand` does. But IMO the implementation of `CopyrightGarbage` should not _enforce_ a "weird" / custom sorting for all its users. So in that regard, I consider the question to be out of scope of this PR.

Reply:
I hadn't notice that the converter actually got dropped by the force push: https://github.com/oss-review-toolkit/ort/compare/8d75b58e60d4e339ec1950026b435a9355c896d1..74804278bcd5b569f0823e96a05b2a0ae5b39471.

Agree that this is out of scope then. I thought the converter was still there which would have changed the sorting of `ImportCopyrightGarbageCommand`. So, then agree it's out of scope. Sorry.